### ISSUE TITLE
Don't display files beginning with a period

### ISSUE THREAD

**Issue by maks** [democloid/picoTracker#29]
"Dotfiles" are by convention not listed on unix systems and should also not be listed in the sample import browser dir listings.

**Comment by democloid** [comment]
@maks this broke with 4ed8fd43fe2f05b5ed889ec4d0a2cd1774b7dd19
Tested working fine before, probably related to file filters used

**Comment by maks** [comment]
> used

Yes definitely I broke it with my change. Ive left it for now as a good first PR for a new contributor to project but I'll fix it if no one gets to it in the next little while 🙂

**Comment by shockdesign** [comment]
@maks Looking at the code here, would you think it would be best to ignore the Dotfiles in the wildcard match function?

**Comment by maks** [comment]
@shockdesign yes that's the change that needs to be made. I left this as a small task for a "good first issue" for new contributors :-)

**Comment by maks** [comment]
done as of #219  though note if file has a `.wav` suffix it will still be shown.
